heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -281,7 +281,7 @@ public static boolean setSchedulerLocation(
 
     Scheduler.SchedulerLocation location = builder.build();
 
-    LOG.log(Level.INFO, "Setting SchedulerLocation: {0}", location);
+    LOG.log(Level.INFO, "Setting Scheduler locations:\n{0}", location);
```

Comment:
Inserting line breaks in log entries can cause pains when reading the logs. Can we remove the \n?